lib/router.js

```diff
@@ -86,6 +88,9 @@ Router.prototype.applyRoutes = function (server) {
       }
 
       self.routes[method].forEach(function (route) {
+        if (typeof route.options.path === 'string' && prefix !== '') {
```

Comment:
Yup. String paths only. When i thought about supporting prefixes, i was also looking at the regex case but I prefer keeping it for string paths only.